src/macros.rs

```diff
@@ -353,6 +355,13 @@ macro_rules! clap_app {
             $($tt)*
         }
     };
+    // Handle multiple attributes in enclosing parentheses
+    (@app ($builder:expr) (@group $name:ident ($($attrs:tt)*) => $($tail:tt)*) $($tt:tt)*) => {
```

Comment:
Let's remove `()` around the attributes. That ways, we don't even need the the above 2 cases for backward compatibility since this would cover both of them.

I was also thinking, we can follow the same syntax from `@arg`, that is make it `@group $name:ident: `.

Reply:
I doubt that's doable. Simply removing the parentheses results in a compiler error: `local ambiguity: multiple parsing options: built-in NTs tt ('attrs') or 1 other option.`

Reply:
Try the `:` option. If you are still getting compiler error, it needs to be worked on for fixing. As of right now, we won't be accepting this API with the parentheses.

Reply:
I think the problem is that we can't seperate the attributes with the following `@arg` definitions without a new syntax: in `$($attr:tt))* => $($tail:tt)*`, the compiler cannot tell which token tree the `=>` should be part of, `$attr` or `$tail` . We can't remove `=>`, too, because that way we won't know which are attributes and which are arg defs.